<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>detailed</class>
 <widget class="QDialog" name="detailed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>431</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>30</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>高分</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>60</y>
     <width>68</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>低分</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="high">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>30</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="low">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>90</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>班级</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="clas">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>90</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>120</y>
     <width>160</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="comboBox">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <item>
       <property name="text">
        <string>数学</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>物理</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>化学</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="searchSection">
      <property name="text">
       <string>区间查询</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Analyze">
      <property name="text">
       <string>单科平均</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="totalAnalyz">
      <property name="text">
       <string>总分平均</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="dataAnalyze">
      <property name="text">
       <string>统计分析</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="sortScore">
        <property name="text">
         <string>排序</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="write">
        <property name="text">
         <string>导出</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
